public_suffix_list.dat

```diff
@@ -13760,4 +13760,11 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Diher Solutions : https://diher.solutions
+// Submitted by Didi Hermawan <didihermawan@aol.com>
+*.diher.solutions
```

Comment:
Need to sort the entries by TLD then SLD in Ascending Alphabetical order (id would come before solutions) in order to comply with the guidelines